library/bignum_mod.c

```diff
@@ -77,8 +77,11 @@ void mbedtls_mpi_mod_modulus_free( mbedtls_mpi_mod_modulus *m )
     switch( m->int_rep )
     {
         case MBEDTLS_MPI_MOD_REP_MONTGOMERY:
-            mbedtls_free( m->rep.mont );
-            break;
+            mbedtls_platform_zeroize( (mbedtls_mpi_uint *) m->rep.mont.rr,
```

Comment:
`mbedtls_platform_zeroize` takes a `void *`, so no need to cast it

Reply:
ah, the need is to cast-away const-ness

Reply:
Casting away constness is highly suspicious and, if actually warranted, requires a comment explaining why it's ok.